spring-cloud-gcp-logging/src/main/java/org/springframework/cloud/gcp/logging/LoggingWebMvcConfigurer.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright 2017 original author or authors.
```

Comment:
I still insist that new classes have to make it into the `master` in the current year.